toy_example/run_pandora_nodenovo.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -eu
+
+singularity pull docker://rmcolq/pandora:latest
+singularity exec ./pandora-latest.simg pandora index prgs/toy_prg.fa
+singularity exec ./pandora-latest.simg pandora compare --genotype -o output_toy_example_no_denovo prgs/toy_prg.fa reads/read_index.tsv
```

Comment:
```suggestion
URI="docker://rmcolq/pandora:latest"
singularity exec "$URI" pandora index prgs/toy_prg.fa
singularity exec "$URI" pandora compare --genotype -o output_toy_example_no_denovo prgs/toy_prg.fa reads/read_index.tsv
```

Reply:
Also, the final `compare` command it missing a value to the `-o` parameter

Reply:
Additionally, the changes I have suggested are more adaptable to singularity version differences. For example, this script fails for me as when I `singularity pull`, the image is named `pandora_latest.sif` not `.simg`

Reply:
> Also, the final `compare` command it missing a value to the `-o` parameter

Not sure I understand this: the value to the `-o` param is `output_toy_example_no_denovo`

Reply:
Argh, I think I will keep doing `singularity pull`, but will check if it is pulled into a `.simg` or `.sif`. Doing `singularity exec` is all fine, I actually preferred it, if it did not take so long to create the container runtime (singularity pull takes ~1min to be done, and then the 2 execs take <1s. If we don't pull before, each exec take >1 min)...

Reply:
Will do `singularity exec` as you mentioned, but reduce it to a single command